README.dk.md

```diff
@@ -0,0 +1,66 @@
+# Sway
+
+Sway er en [i3](https://i3wm.org/)-kompatibel [Wayland](http://wayland.freedesktop.org/) typograf.
+Læs [Ofte stillede spørgsmål](https://github.com/swaywm/sway/wiki).
+Deltag på [IRCkanalen](http://webchat.freenode.net/?channels=sway&uio=d4) (#sway på irc.freenode.net).
+
+Hvis du har lyst til at støtte sway udviklingen, venlist send bidrag på [SirCmpwn'sPatreon side](https://patreon.com/sircmpwn).
+
+## Udgivelses Signaturer
+
+Udgivelser er signeret med [B22DA89A](http://pgp.mit.edu/pks/lookup?op=vindex&search=0x52CB6609B22DA89A)
+og publiseret på [GitHub](https://github.com/swaywm/sway/releases).
+
+## Installation
+
+### Fra Pakker
+
+Sway er tilgængelig i mange distributioner. Prøv at installere pakken "svaje" til
```

Comment:
```suggestion
Sway er tilgængelig i mange distributioner. Prøv at installere pakken "sway" til
```
I assume you didn't intend to translate sway :)

Reply:
indeed i did not :)